src/priorityQueue.js

```diff
@@ -49,13 +49,24 @@ export default class PriorityQueue {
 
   dequeue() {
     if (this.size === 0) return null
-    let element = this.elements.front()
-    this.elements.pop(element)
-    return element
+    let dequeueData = this.front()
```

Comment:
Consider making these identifier names more concise (since they are in the `dequeue` function, it is clear that they are related to `dequeue` already): `data`, `element`, `index`.